packages/normalizr/src/entities/Entity.ts

```diff
@@ -331,7 +330,8 @@ First three members: ${JSON.stringify(input.slice(0, 3), null, 2)}`;
     this: T,
     input: Readonly<Partial<AbstractInstanceType<T>>>,
     unvisit: schema.UnvisitFunction,
-  ): [AbstractInstanceType<T>, boolean, boolean] {
+  ): [denormalized: AbstractInstanceType<T>, found: boolean, suspend: boolean] {
```

Comment:
```suggestion
  ): [denormalized: AbstractInstanceType<T>, found: boolean, suspend: boolean] as const {
```
?

Reply:
As const is for values but this is a type Def so it's already a tuple

Reply:
ah ok, TIL